### DISCUSSION TITLE
How do I create a GitHub link?

### DISCUSSION THREAD

**Discussion by jupiterwebdev** [community/community#37000] [Other Feature Feedback, Questions, & Ideas] [answered]
How do I create a GitHub link?

**Comment by radar-blip** [comment]
Depends.  A link to what?

One example: `[Github](https://github.com/)` = [Github](https://github.com/)

**Reply by DSDmark** [reply]
Hey @mykyta777,

If you want **Github-link** of your profile, then you can create like this:- 

```https://github.com/mykyta777```
>  In other words 

```https://github.com/{username}```

And you can create short links as @linuxlawson describe if you want to write in whatever place you want :-

```[**mykyta777**](https://github.com/mykyta777)``` = [**mykyta777**](https://github.com/mykyta777)

**Reply by AkinolaRuth** [reply]
https://github.com/LadyDEmpire is showing 404 error

**Comment by ra0321** [comment, marked as answer]
n GitHub.com, navigate to the main page of the repository.
Locate the code you'd like to link to: ...
Choose whether to select a single line or a range: ...
To the left of the line or range of lines, click . ...
Navigate to the conversation where you want to link to the code snippet.
